modules/menu/menu.c

```diff
@@ -678,6 +678,13 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 
 	switch (ev) {
 
+	case UA_EVENT_CALL_ARRIVED:
+		if (menu.dnd) {
+			call_set_rejected(call, 480,
+					  "Temporarily Unavailable");
+		}
```

Comment:
is it possible to decline the call directly here, instead of setting it to rejected and replying later ?

Reply:
here we don't have the `sip_msg`